test/spec.index.js

```diff
@@ -1076,6 +1076,153 @@ describe('Template Mixins', function () {
+                middleware(req, res, next);
+                res.locals['time']().call(res.locals, '12:00pm 26 March 2015').should.equal('Midday 26 March 2015');
+            });
+
+            it('should pass through other times', function () {
+                middleware(req, res, next);
+                res.locals['time']().call(res.locals, '6:30am 26 March 2015').should.equal('6:30am 26 March 2015');
+            });
+        });
+
+        describe('uppercase', function () {
+            beforeEach(function () {
+                middleware = mixins();
+            });
+
+            it('adds a function to res.locals', function () {
+                middleware(req, res, next);
+                res.locals['uppercase'].should.be.a('function');
+            });
+
+            it('returns a function', function () {
+                middleware(req, res, next);
+                res.locals['uppercase']().should.be.a('function');
+            });
+
+            it('changes text to uppercase', function () {
+                middleware(req, res, next);
+                res.locals['uppercase']().call(res.locals, 'abcdEFG').should.equal('ABCDEFG');
+            });
+
+            it('returns an empty string if no text given', function () {
+                middleware(req, res, next);
+                res.locals['uppercase']().call(res.locals).should.equal('');
+            });
+        });
+
+        describe('lowercase', function () {
+            beforeEach(function () {
+                middleware = mixins();
+            });
+
+            it('adds a function to res.locals', function () {
+                middleware(req, res, next);
+                res.locals['lowercase'].should.be.a('function');
+            });
+
+            it('returns a function', function () {
+                middleware(req, res, next);
+                res.locals['lowercase']().should.be.a('function');
+            });
+
+            it('changes text to uppercase', function () {
+                middleware(req, res, next);
+                res.locals['lowercase']().call(res.locals, 'abcdEFG').should.equal('abcdefg');
+            });
+
+            it('returns an empty string if no text given', function () {
+                middleware(req, res, next);
+                res.locals['lowercase']().call(res.locals).should.equal('');
+            });
+        });
+
+        describe('currency', function () {
+            beforeEach(function () {
+                middleware = mixins();
+            });
+
+            it('adds a function to res.locals', function () {
+                middleware(req, res, next);
+                res.locals['currency'].should.be.a('function');
+            });
+
+            it('returns a function', function () {
+                middleware(req, res, next);
+                res.locals['currency']().should.be.a('function');
+            });
+
+            it('correctly formats 3.00', function () {
```

Comment:
I don't think this is very descriptive. What about something like 'formats whole numbers without decimal places'?